Felienne/hedy
# [FEATURE] Create interactive tutorial for new users

**Description**
In this PR we add an exciting new functionality: An interactive tutorial! The Hedy interface has become a bit crowded with new features and functionalities, making it harder for new users to understand our interface. We aim to tackle this issue by creating an interactive tutorial that is automatically shown to new users. Making all UI elements visible one by one and explaining their functionality. We also add two achievements for both the student and teacher tutorial.

**Fixes**
This PR fixes #2679 and fixes #2703.

**How to test**
Make sure you are logged in and then navigate to `/tutorial` and walkthrough the different steps. Another option is to create a new account, login and get re-directed to the landing page. This page is updated as well and select the "start tutorial" option. To view this page you can also simply login and navigate to `/landing-page`. Make sure you are a non-teacher account in this case! Otherwise the landing-page will only show the teacher tutorial option.

For the teacher tutorial follow the same steps, make sure you are logged in and navigate to the landing page. Another option is to directly navigate to `/teacher-tutorial`. Verify that all steps are working, also in combination with the newly added achievements.
